test/unittests/commands/project/test_models_project.py

```diff
@@ -0,0 +1,54 @@
+from test.unittests.commands.xmlupload.connection_mock import ConnectionMockBase
+
+import pytest
+
+from dsp_tools.commands.project.models.project import Project
+from dsp_tools.models.langstring import LangString, Languages
+
+
+@pytest.fixture()
+def project() -> Project:
+    return Project(
+        con=ConnectionMockBase(),
+        iri="http://rdfh.ch/test",
+        shortcode="0FF0",
+        shortname="test_project",
+        longname="Test Project",
+        description=LangString({Languages.EN: "This is a test project", Languages.DE: "Das ist ein Testprojekt"}),
+        keywords=set(),
+        selfjoin=False,
+        status=True,
+        logo="logo.gif",
+    )
+
+
+def test_return_values(project: Project) -> None:
+    assert project.iri == "http://rdfh.ch/test"
+    assert project.shortcode == "0FF0"
+    assert project.shortname == "test_project"
+    assert project.longname == "Test Project"
+    assert project.description["en"] == "This is a test project"
+    assert project.description["de"] == "Das ist ein Testprojekt"
+    assert project.selfjoin is False
+    assert project.status is True
+    assert project.keywords == set()
```

Comment:
The reason why I tested it, was because it originally was, and the variables are not accessed directly but through functions with `@property` decorators. Do you think this does not need to be tested?

Reply:
makes sense. To me this seems excessive, but admittedly, with properties, this could in theory break (unlike dataclasses), so it's also ok to have this test here. And it doesn't hurt us either.